packages/react/src/primitives/TextField/TextField.tsx

```diff
@@ -26,6 +27,8 @@ export const TextField: React.FC<TextFieldProps> = ({
   justifyContent,
   label,
   labelHidden = false,
+  hideShowPassword = true,
```

Comment:
Do we want to have this on the TextField rather than just the PasswordField?

Reply:
Yes, I must have forgotten to remove it. This one doesn't do anything anymore.